addons/mrp_account/i18n/vi.po

```diff
@@ -92,7 +92,7 @@ msgid ""
 "bill of materials, for manufactured products only."
 msgstr ""
 "Tính giá của sản phẩm sử dụng các sản phẩm và hoạt động của các định mức "
-"nguyên vật liệu liên quan, chỉ dành cho các sản phẩm có thể sản xuất được."
+"nguyên liệu liên quan, chỉ dành cho các sản phẩm có thể sản xuất được."
```

Comment:
xem lại toàn hệ thống xem thuật ngữ bill of materials đang dịch là gì. Cần thống nhất trên toàn hệ thống đối với các thuật ngữ.